### DISCUSSION TITLE
lil mistake in the component class

### DISCUSSION THREAD

**Discussion by ayp234** [FZJ-IEK3-VSA/FINE#57] [Q&A]
while I was searching for "TAC" in the code of the component class, I came across this supposed mistake: 
![grafik](https://github.com/FZJ-IEK3-VSA/FINE/assets/82814964/611a09f2-73b6-4fbc-9dab-330de378ac71)

**Comment by JohannesBehrens** [comment]
Thanks for your comment. This issue will be fixed soon.
